source/chapters/configuration.rst

```diff
@@ -79,20 +79,42 @@ inputs and outputs to be used.
   This is useful for setups where you can only hear the right or the left
   speaker.
 
-* **Microphone/Talkover Mix**: This setting controls whether microphone outputs
-  are mixed into the Master output or only into the broadcasting output. This is
-  useful for preventing echo in broadcasting setups.
+* **Microphone Mix Mode**: Controls the routing of microphone inputs. Refer to
+  SECTION below for more details.
```

Comment:
Yes, that's just a TODO for putting an actual link to the section there. I wrote that before actually writing out the section it will link to.